Tasks/XamarinTestCloud/Strings/resources.resjson/ko-KR/resources.resjson

```diff
@@ -26,6 +26,6 @@
   "loc.input.label.testCloudLocation": "test-cloud.exe 위치",
   "loc.input.label.optionalArgs": "선택적 인수",
   "loc.input.help.optionalArgs": "추가 인수를 test-cloud.exe에 전달했습니다.",
-  "loc.input.label.publishNUnitResults": "결과를 VSO/TFS에 게시",
-  "loc.input.help.publishNUnitResults": "선택하면, --nunit-xml 옵션이 test-cloud.exe에 전달됩니다. NUnit xml 파일의 결과는 VSO/TFS에 게시됩니다."
+  "loc.input.label.publishNUnitResults": "결과를 TFS/Team Services에 게시",
```

Comment:
I double-checked the file.  This appears to just be a diff error from GitHub.  Thanks!